ai_ta_backend/main.py

```diff
@@ -517,9 +517,11 @@ def ingest_canvas():
 @app.route('/getNomicMap', methods=['GET'])
 def nomic_map():
   course_name: str = request.args.get('course_name', default='', type=str)
-  if course_name == '':
+  map_type: str = request.args.get('map_type', default='', type=str)
```

Comment:
Added map_type here to enable map fetching for both coversation and document maps.